ITEST/itest.storelogdata/test/org/openl/itest/RunStoreLogDataITest.java

```diff
@@ -243,149 +270,109 @@ public void testKafkaMethodServiceOk() throws Exception {
             .until(validateH2(values), equalTo(true));
     }
 
-    @Test
-    public void testKafkaServiceOkWithNoOutputTopic() throws Exception {
-        final String REQUEST = "{\"hour\": 5}";
-
-        truncateCassandraTableIfExists(KEYSPACE, DEFAULT_TABLE_NAME);
-        truncateH2TableIfExists(DEFAULT_H2_TABLE_NAME);
-
-        KeyValue<String, String> record = new KeyValue<>(null, REQUEST);
-        record.addHeader(KafkaHeaders.METHOD_NAME, HELLO_METHOD_NAME, StandardCharsets.UTF_8);
-        cluster.send(SendKeyValues.to("hello-in-topic-5", Collections.singletonList(record)).useDefaults());
-
-        ExpectedLogValues values = new ExpectedLogValues(REQUEST,
-            null,
-            HELLO_METHOD_NAME,
-            SIMPLE5_SERVICE_NAME,
-            KAFKA_PUBLISHER_TYPE);
-        values.setResponseProvided(true);
-        given().ignoreException(InvalidQueryException.class)
-            .await()
-            .atMost(AWAIT_TIMEOUT, TimeUnit.SECONDS)
-            .until(validateCassandra(values), equalTo(true));
-
-        given().ignoreExceptions()
-            .await()
-            .atMost(AWAIT_TIMEOUT, TimeUnit.SECONDS)
-            .pollInterval(POLL_INTERVAL_IN_MILLISECONDS, TimeUnit.MILLISECONDS)
-            .until(validateH2(values), equalTo(true));

```

Comment:
Coverage by the tests should be kept or improved. Do not delete tests without providing valid reasons.